src/daily-dropoffs-report.tsx

```diff
@@ -0,0 +1,376 @@
+import { VNode } from "preact";
+import { useContext, useEffect, useRef, useState } from "preact/hooks";
+import { utcDays } from "d3-time";
+import { useQuery } from "preact-fetching";
+import { format } from "d3-format";
+import { group, ascending, max } from "d3-array";
+import { scalePoint, scaleLinear, scaleOrdinal, NumberValue } from "d3-scale";
+import { csvParse, autoType } from "d3-dsv";
+import { line as d3Line } from "d3-shape";
+import { select } from "d3-selection";
+import { Axis as D3Axis, axisBottom, axisLeft } from "d3-axis";
+import { schemeCategory10 } from "d3-scale-chromatic";
+import { path as reportPath } from "./report";
+import { ReportFilterContext } from "./context/report-filter-context";
+import Table, { TableData } from "./table";
+import { AgenciesContext } from "./context/agencies-context";
+
+// enum Mode {
+//   /**
+//    * Starts funnel at the welcome screen
+//    */
+//   OVERALL = "overall",
+//   /**
+//    * Starts funnel at the image submission screen
+//    */
+//   BLANKET = "blanket",
+// }
+
+enum Step {
```

Comment:
I can't figure out what the source of this eslint error is... this is the first place that the enum is declared...

```
   29:6   error  'Step' is already declared in the upper scope on line 29 column 6  no-shadow
```

Reply:
solution: a3f3a9e50a3f7d020ee8fa6ad1e54d6f27b48d78, h/t @aduth and https://github.com/typescript-eslint/typescript-eslint/issues/2483#issuecomment-687095358